lib/util.js

```diff
@@ -1,70 +1,103 @@
 import fs, { close, openSync, statSync, utimesSync, accessSync } from 'node:fs'; // need import fs here due to test stubbing
 import util from 'node:util';
 import { EOL } from 'node:os';
-import _ from 'lodash';
 import gitUrlParse from 'git-url-parse';
 import semver from 'semver';
 import osName from 'os-name';
+import { Eta } from 'eta';
 import Log from './log.js';
 
+const debug = util.debug('release-it:shell');
+
+const wait = ms => new Promise(resolve => setTimeout(resolve, ms));
+const before = (n, func) => {
+  var result;
+  if (typeof func != 'function') {
+    throw new TypeError('Missing argument for `func`');
+  }
+  n = parseInt(n);
+  return function () {
+    if (--n > 0) {
+      result = func.apply(this, arguments);
+    }
+    if (n <= 1) {
+      func = undefined;
+    }
+    return result;
+  };
+};
+const tryStatFile = filePath => {
+  try {
+    return statSync(filePath);
+  } catch (e) {
+    debug(e);
+    return null;
+  }
+};
+
 export const execOpts = {
   stdio: process.env.NODE_DEBUG && process.env.NODE_DEBUG.indexOf('release-it') === 0 ? 'pipe' : []
 };
 
-const debug = util.debug('release-it:shell');
-
-const readJSON = file => JSON.parse(fs.readFileSync(file, 'utf8'));
+export const readJSON = file => JSON.parse(fs.readFileSync(file, 'utf8'));
 
 const pkg = readJSON(new URL('../package.json', import.meta.url));
 
-const log = new Log();
-
-const getSystemInfo = () => {
+export const getSystemInfo = () => {
   return {
     'release-it': pkg.version,
     node: process.version,
     os: osName()
   };
 };
 
-const format = (template = '', context = {}) => {
+export const format = (template = '', context = {}) => {
+  if (!context || context === null || !template || template === null || template.indexOf('${') === -1) return template;
+
   try {
-    return _.template(template)(context);
+    const eta = new Eta({
+      autoEscape: false,
+      useWith: true,
+      tags: ['${', '}'],
+      parse: { interpolate: '' },
+      rmWhitespace: false,
+      autoTrim: false
+    });
+    return eta.renderString(template, context);
   } catch (error) {
+    const log = new Log();
```

Comment:
Why is this moved here, would also expect this at module scope?

Reply:
with the current setup, i get `ReferenceError: Cannot access 'Log' before initialization` when moving it to module scope. Since it is only used there, I see no reason to move it away from where it is used